projects/components/src/search-box/search-box.component.ts

```diff
@@ -46,6 +46,9 @@ export class SearchBoxComponent implements OnInit, OnChanges {
   @Input()
   public displayMode: SearchBoxDisplayMode = SearchBoxDisplayMode.Border;
 
+  @Input()
```

Comment:
We already have usecases. We don't want search usecases inside multi-select, select and other client side filtering needs to be trigger based.

Reply:
I left the FF logic out of the component so that the consumer has complete flexibility to pick the mode (based on one or more FFs or other parameters) and also to keep a widely used basic component as clean as possible.

Reply:
I buy the select cases, but client side or not is an impl detail and they should have the same UX if it's the same control. Leaving the input with flag controlled default behavior gives the same functionality as you describe, it's just much easier to use on the typical consumer.